CODING_STYLE

```diff
@@ -42,8 +42,8 @@
   "log_oom()" for then printing a short message, but not in "library" code.
 
 - Do not issue NSS requests (that includes user name and host name
-  lookups) from PID 1 as this might trigger deadlocks when those
-  lookups involve synchronously talking to services that we would need
+  look-ups) from PID 1 as this might trigger deadlocks when those
+  look-ups involve synchronously talking to services that we would need
```

Comment:
"look-up" might be more correct according to a dictionary, but in CS everybody says "lookup". (Like "login", and many other things.) The same for runtime.